src/autocomplete.js

```diff
@@ -136,6 +135,15 @@
             else{
                 this.options.selectFirstMatch = false
             }
+
+            var attr = this.$input.attr("validation-function")

```

Comment:
I think it would be better if the attribute's name was `validation`. That way it would be the same as the property in the options object (see line 350).

Reply:
yeaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaah